source/reference/minio-mc/mc-find.rst

```diff
@@ -106,6 +108,22 @@ Parameters
 
    Limits directory navigation to the specified depth.
 
+.. mc-cmd:: --metadata
+   :optional:
+
+   .. versionadded:: mc RELEASE.2023-04-12T02-21-51Z
+
+   **For use with MinIO deployments only.**
+
+   Match files with metadata that matches a specified ``key=value``.
```

Comment:
I'm not totally following `files` vs `objects` in this section. The existing text uses `objects on MinIO` or `objects on a filesystem` in several places, does that map to `objects` and `files` in this text? 

It's not clear to me if `--metadata` is for files, or both files and objects.

Reply:
Objects. Thanks for catching this.